contract/package.json

```diff
@@ -30,7 +30,7 @@
     "@agoric/zone": "^0.2.2",
     "@cosmjs/amino": "^0.32.3",
     "@cosmjs/proto-signing": "^0.32.3",
-    "@endo/eslint-plugin": "^0.5.2",
+    "@endo/eslint-plugin": "^2.2.0",
```

Comment:
I'm still learning... how did you choose that version number? is it just the current in npm?

Did anything in particular prompt changing it? Should we generally recommend using the then-current versions of all lint-plugins in https://github.com/Agoric/agoric-sdk/discussions/8274 ?

Reply:
for this one I used `yarn upgrade-interactive`